tests/integration/incremental_unique_id_test/test_incremental_unique_id.py

```diff
@@ -0,0 +1,270 @@
+from dbt.exceptions import ParsingException
+from tests.integration.base import DBTIntegrationTest, use_profile
+from collections import namedtuple
+from pathlib import Path
+
+
+TestResults = namedtuple(
+    'TestResults',
+    ['seed_count', 'model_count', 'seed_rows', 'inc_test_model_count',
+     'opt_model_count', 'relation'],
+)
+
+
+class TestIncrementalUniqueKey(DBTIntegrationTest):
+    @property
+    def schema(self):
+        return 'incremental_unique_key'
+
+    @property
+    def models(self):
+        return 'models'
+
+    def count_test_relations(self, seed, incremental_model):
+        '''Idempotently create some number of seeds and incremental models'''
+        seed_result_set = self.run_dbt(
+            ['seed', '--select', seed, '--full-refresh']
+        )
+        model_result_set = self.run_dbt(
+            ['run', '--select', incremental_model, '--full-refresh']
+        )
+        return len(seed_result_set), len(model_result_set)
+
+    def get_seed_rows_after_update(self, seed, update_sql_file):
```

Comment:
Same with this one, it doesn't have to be it's own function. But more of a nitpicking here

Reply:
Same remark about focusing on the in-code effect of knowing what the updates are doing rather than focusing on the updates themselves which are database side effects.